javascripts/enterEventListener.js

```diff
@@ -0,0 +1,26 @@
+const addToArray = require('./addToArray');
+const buildDomString = require('./dom');
+const data = require('./data');
+const deleteEventListener = require('./deleteEventListener');
+const edit = require('./editEventListener');
+
+const enterListener = () => {
+  console.log('hello');
+  document.querySelector('input').addEventListener('keypress', keypressEnter);
+};
+
+const keypressEnter = (e) =>
+{
+  const key = e.keyCode;
+  if (key === 13)
+  {
+    const messages = data.getMessages();
```

Comment:
if .getMessages() is called here then it will not contain the new message object since addToArray() is called after getting the data